stdlib/private/StdlibCollectionUnittest/LoggingWrappers.swift.gyb

```diff
@@ -556,11 +556,15 @@ public struct Logging${Kind}<Base: ${Kind}Type> : ${Kind}Type, LoggingType {
     return base._copyToNativeArrayBuffer()
   }
 
-  /// Copy a Sequence into an array.
-  public func _initializeTo(ptr: UnsafeMutablePointer<Base.Generator.Element>)
-    -> UnsafeMutablePointer<Base.Generator.Element> {
+  /// Copy `self` into uninitialized contiguous memory of size `capacity`,
+  /// returning one past the last element initialized, or a `nil` pointer
+  /// if the copy cannot be safely made given `capacity`.
+  public func _initializeTo(
```

Comment:
I really don’t like the way uses of this new function signature reads.
Consider re-casting it [the way it will soon be renamed](https://github.com/apple/swift/blob/swift-3-api-guidelines/stdlib/public/core/Collection.swift#L594) anyway.  I don’t understand what `capacity` is supposed to be.  What am I allowed/required to pass here?